### PR TITLE
Added TileDB as well as COGs as a cloud native dataset example

### PR DESCRIPTION
**Pull request by normanb**
I have added TileDB as an example of a cloud native dataset and changed the wording around COGs to acknowledge that there are other formats that be used. TileDB has a format driver within GDAL/PDAL and can be used with the pointcloud extension as well as with EO, SAR, datacube etc

### CODE REVIEW

File: overview.html [hunk 1]
```diff
@@ -41,7 +41,7 @@ <h1>SpatioTemporal Asset Catalogs In Depth</h1>
...
+							<p>The STAC spec itself provides a lowest common denominator JSON format to wrap around any relevant data about the earth. The core <a href="http://geojson.org">GeoJSON</a> object and related structures are designed for extension, so it can adapt to different domains. For more information about the specification, see the <a href="core.html">Spec Overview</a> page. The core JSON pages can be transformed into browsable, interactive HTML pages with tools like <a href="https://github.com/radiantearth/stac-browser/">STAC Browser</a>. When STAC is used in concert with emerging formats like <a href="http://cogeo.org"</a>Cloud Optimized GeoTIFF</a> or <a href="https://github.com/TileDB-Inc/TileDB/">TileDB</a></href>the result vastly lowers the barriers for anybody to find and use geospatial assets like satellite imagery:</p>
```
Comment (m-mohr) [inline overview.html:44]: That `</href>` is strange...

Comment (normanb) [reply overview.html:44]: It is, I will remove that. I noticed that the anchor for COG is also odd, I will edit that one as well.

**Comment by cholmes** [comment]
I'm happy to add this, but is there any way to link to 'tiledb arrays' - like the storage format (not sure if I have the name exactly right). Without having talked to you I would be confused as to why STAC is referring to a database or 'storage engine'. 

https://github.com/TileDB-Inc/TileDB/blob/dev/format_spec/FORMAT_SPEC.md seems to be the reference I can find, which is a little sparse (could use an intro / problem statement or something), but for people coming from STAC without knowing TileDB I think that would make more intuitive sense.

**Comment by normanb** [comment]
:+1: @cholmes I have linked to the format spec and we (TileDB Inc) can expand the format spec to include an intro.

**Comment by cholmes** [comment]
Awesome, thanks!

**Pull request merged**